README.md

```diff
@@ -1,22 +1,22 @@
-## [PORTFOLIO](https://khaletska.github.io/portfolio/)
+### You can find my full portfolio [here](https://khaletska.github.io/portfolio/)
```

Comment:
I didn't notice your porftolio link until I noticed it is an actual hyperlink. If you feel it is prominent and visible then let's no make this change

Reply:
Let's leave it as it was